### PR TITLE
Option to New GitHub Pull Tester

### PR DESCRIPTION
**Pull request by luizeof**
Add ENV to new Github Pull Tester https://github.com/mautic/mautic-tester

### CODE REVIEW

File: apache/docker-entrypoint.sh [hunk 1]
```diff
@@ -14,6 +14,13 @@ if [ -n "$MYSQL_PORT_3306_TCP" ]; then
         fi
 fi
 
+# Github Pull Tester
+if [ -n "$MAUTIC_TESTER" ]; then
+  echo >&2 "Copying Mautic Github Pull Tester"
+  git clone https://github.com/mautic/mautic-tester.git /var/www/tester
+  cp -rf /var/www/tester/tester.php /var/www/html/tester.php
```
Comment (Gregy) [inline apache/docker-entrypoint.sh]: This seems needlessly complicated to me. What about simply using wget here?
```
wget https://raw.githubusercontent.com/mautic/mautic-tester/master/tester.php
```

File: fpm/docker-entrypoint.sh [hunk 2]
```diff
@@ -14,6 +14,13 @@ if [ -n "$MYSQL_PORT_3306_TCP" ]; then
         fi
 fi
 
+# Github Pull Tester
+if [ -n "$MAUTIC_TESTER" ]; then
+  echo >&2 "Copying Mautic Github Pull Tester"
+  git clone https://github.com/mautic/mautic-tester.git /var/www/tester
+  cp -rf /var/www/tester/tester.php /var/www/html/tester.php
+fi
+
```
Comment (Gregy) [inline fpm/docker-entrypoint.sh]: Could you please add the same changes to the common/docker-entrypoint.sh?

**Comment by luizeof** [comment]
Thanks for Feedback @Gregy

**Comment by Gregy** [comment]
I tested it but it doesn't work. It is missing wget executable. Can you add it to the Dockerfile?

**Comment by luizeof** [comment]
@Gregy ready for test

**Comment by luizeof** [comment]
@Gregy I´ve put the downloader on line 128 ... after mautic downloaded

**Comment by Gregy** [comment]
Great. Thank you.

**Pull request merged**